tests/test_dummy.py

```diff
@@ -0,0 +1,3 @@
+import sqlalchemy
```

Comment:
Pretty useless obviously, if you find the time, try to write some basic tests, shouldn't be too hard.